bot.py

```
@@ -98,30 +98,25 @@ def load_data():
     return {'hackmd': None, 'connpass': None}
 
 def save_data(data):
-    """Google Sheetsまたはローカルファイルにデータを保存"""
+    """Google Sheetsとローカルファイルの両方にデータを保存"""
+    # ローカルファイルに常に保存（フォールバック時の整合性を保証）
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)